app/assets/javascripts/discourse/tests/acceptance/sidebar-plugin-api-test.js

```diff
@@ -54,6 +57,10 @@ acceptance("Sidebar - section API", function (needs) {
                 },
               ];
             }
+            @bind
+            willDestroy() {
+              window.section_destroy = "section test";
```

Comment:
O we generally do not want to rely on a global here. You can just declare a variable that is scoped to this file and set it instead.